ee/DEV.md

```diff
@@ -0,0 +1,37 @@
+## Getting it up and running
+`docker-compose -f ee/docker-compose.ch.yml up`
+
+## Fixing broken frontend build
+`docker run -it ee_web yarn build`
+
+## Running Python + Webpack locally
+- Run all the services
+  - Stop local kafka, clickhouse and zookeeper instances if you have them
+  - Same for redis, though it doesn't really matter much
+  - `docker-compose -f ee/docker-compose.ch.yml up db redis zookeeper kafka clickhouse`
+  - Add to `/etc/hosts`:
+    - `127.0.0.1       kafka` (setting KAFKA_URL later doesn't seem to work)
```

Comment:
nicely done. This is a bug/feature with how the consumer picks up hosts for kafka from zookeeper.